src/api/spec/features/beta/webui/login_spec.rb

```diff
@@ -31,29 +31,29 @@
 
   scenario 'login via widget' do
     visit root_path
-    within('#navigation') do
+    within(desktop? ? '#navigation' : '#bottom-navigation') do
```

Comment:
Evokes the question why this is called #bottom-navigation on mobile...

Reply:
Right, this something should change in the layout. I prefer to do the change in a different PR.